assets/openldap/values.yaml

```diff
@@ -1,7 +1,7 @@
 name: ldap
 image: osixia/openldap:1.5.0
 tls:
-  enabled: false
+  enabled: true
```

Comment:
will this impact the rbac examples, do those have ldap or ldaps?

Reply:
it enables both modes - so no impact